recipes/nceplibs-ip/bld.bat

```diff
@@ -0,0 +1,24 @@
+
+mkdir build
+cd build
+
+REM Make shared libs
+cmake -G"Ninja" ^
+      %CMAKE_ARGS% ^
+      -DCMAKE_BUILD_TYPE=Release ^
+      -DCMAKE_PREFIX_PATH=%PREFIX% ^
+      -DCMAKE_INSTALL_PREFIX=%PREFIX% ^
+      -DCMAKE_FIND_FRAMEWORK=NEVER ^
+      -DCMAKE_FIND_APPBUNDLE=NEVER ^
+      -DBUILD_SHARED_LIBS=ON ^
+      -DOPENMP=ON ^
+      %SRC_DIR%
+if errorlevel 1 exit 1
+
+REM Build step
+cmake --build . --config Release
+if errorlevel 1 exit 1
+
```

Comment:
Is there are `make check` or `ctest` w can run here?

Reply:
You can perform `make test` to run the CMake tests.

Reply:
There is already a "ctest ..." in build.sh that runs all the tests - I made sure they all passed on my local Linux machine before submitting.

I can add a similar "ctest ..." command for Windows, but can only run the bld.bat script by committing.

I'll put together a commit in a little bit.

Reply:
The "if" block that is in the "build.sh" to run the "ctest..." line I got from an example somewhere.  Probably just have the "ctest..." command directly in the "bld.bat".

Reply:
Well, you are skipping Windows. I ended up reviewing this file just b/c it was the first when scrolling.

Reply:
I thought I might have skipped Windows - but I was confused.  If I skipped Windows, why did a Windows check run?
https://github.com/conda-forge/staged-recipes/pull/22659/checks

Not complaining, just wondering.

Reply:
> I thought I might have skipped Windows - but I was confused. If I skipped Windows, why did a Windows check run?

Skipping the build doesn't skip the CI here. The CI will always be green though b/c it just starts, check the recipe, and skips it.